praw/objects.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, reddit_session, json_dict=None, fetch=True,
         self.reddit_session = reddit_session
         self._underscore_names = underscore_names
         self._uniq = uniq
-        self.has_fetched = self._populate(json_dict, fetch)
+        self._has_fetched = self._populate(json_dict, fetch)
```

Comment:
Changing this would be a breaking change requiring a major version bump (without warning). What I suggest is that we add a `has_fetched` method that is decorated both by `@property`, and `@deprecated` that simply returns the result of `_has_fetched`. The deprecation warning should state that attribute will be removed in PRAW4, and then we simply clean it up when working on PRAW4.